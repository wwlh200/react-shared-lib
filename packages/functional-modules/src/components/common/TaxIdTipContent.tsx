import * as React from 'react';
import InfoTooltip from './InfoTooltip';

export class TaxIdTipContent extends React.PureComponent {
  render() {
    return (
      <React.Fragment>
        <span>test</span>
      </React.Fragment>
    );
  }
}

export default InfoTooltip(TaxIdTipContent);
