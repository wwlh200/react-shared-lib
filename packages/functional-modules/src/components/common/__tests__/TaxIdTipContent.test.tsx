import * as React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import { TaxIdTipContent } from '../TaxIdTipContent';

describe('TaxesTipContent', () => {
  it('should render TaxesTipContent', () => {
    const component = shallow(<TaxIdTipContent/>);
    expect(shallowToJson(component)).toMatchSnapshot();
  });
});
