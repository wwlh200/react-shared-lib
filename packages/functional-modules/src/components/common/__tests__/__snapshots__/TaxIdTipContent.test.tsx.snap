// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaxesTipContent should render TaxesTipContent 1`] = `
<Fragment>
  <p>
    <LocalTranslate
      locKey="paymentInfo.taxId.tooltip.examples_include"
    />
  </p>
  <ul>
    <li>
      <LocalTranslate
        locKey="paymentInfo.taxId.tooltip1"
      />
    </li>
    <li>
      <LocalTranslate
        locKey="paymentInfo.taxId.tooltip2"
      />
    </li>
    <li>
      <LocalTranslate
        locKey="paymentInfo.taxId.tooltip3"
      />
    </li>
    <li>
      <LocalTranslate
        locKey="paymentInfo.taxId.tooltip4"
      />
    </li>
    <li>
      <LocalTranslate
        locKey="paymentInfo.taxId.tooltip5"
      />
    </li>
    <li>
      <LocalTranslate
        locKey="paymentInfo.taxId.tooltip6"
      />
    </li>
  </ul>
</Fragment>
`;
